dashboard/src/components/DeploymentFormBody/BasicDeploymentForm/CustomFormParam.test.tsx

```diff
@@ -0,0 +1,36 @@
+import { mount } from "enzyme";
+import { IBasicFormParam } from "shared/types";
+import CustomFormComponentLoader from "./CustomFormParam";
+
+const param = {
+  path: "enableMetrics",
+  value: true,
+  type: "boolean",
+  customComponent: {
+    className: "test",
+  },
+} as IBasicFormParam;
+
+const defaultProps = {
+  param,
+  handleBasicFormParamChange: jest.fn(),
+};
+
+// Mocking the window so that the injected components are imported correctly
+const { location } = window;
+beforeAll((): void => {
+  // @ts-ignore
+  delete window.location;
+  // @ts-ignore
+  window.location = {
+    origin: "./fixtures",
+  };
+});
+afterAll((): void => {
+  window.location = location;
+});
+
+it("should render a custom form component", () => {
+  const wrapper = mount(<CustomFormComponentLoader {...defaultProps} />);
+  expect(wrapper).toMatchSnapshot();
```

Comment:
rather than a snapshot I would just check if `CustomComponent` exists.

Reply:
@aanthonyrizzo I think you missed this comment.

Reply:
yes I did, my bad